tap_github/schema_objects.py

```diff
@@ -19,6 +19,18 @@
     th.Property("site_admin", th.BooleanType),
 )
 
+# The actor object is common through GraphQL
+actor_object = th.ObjectType(
```

Comment:
I think in previous graphql prototyping we've re-used the user_object even for graphQL:
- renamed the http_url value to html_url in the graphQL query
- omitted the resource_path from the query
- gravatar_url isn't available from the graphQL API but I think it will just populate nulls in that column if you use user_agent in your schema (needs a sanity check)